### PR TITLE
chore(writing): write an awesome readme

### PR DESCRIPTION
**Pull request by kymppi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+
+1. Fork the repository
+1. Make your changes
+1. Test that those work 😄
+1. Push the changes to your fork and make a PR (Pull Request)
+1. Now someone will review it and hopefully merge it to the repository 🎉🎉
+
+## Support
+
+For support, join [Novu's Discord server](https://discord.gg/novu) or email [me@midka.dev](mailto:me@midka.dev).
```
Comment (kymppi) [inline README.md:93]: I can be the maintainer of this `crate` (package in the npm world)

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+        println!(
+            "Notification sent!!! \n\nack: {}\nstatus: {}\ntransaction_id: {}",
+            event.acknowledged, event.status, event.transaction_id
+        )
+    }
+    Err(api_error) => println!("An error occurred: {}", api_error),
+}
+```
+
+#### Things you need to done
```
Comment (scopsy) [inline README.md:65]:
```suggestion
#### Things you need to do
```

File: README.md (hunk 1 above)
Comment (scopsy) [reply README.md:93]: Sounds like a plan!

**Review by scopsy — APPROVED** [review]
Looks great, added a small typo, but other than that feel free to merge!

File: README.md (hunk 2 above)
Comment (kymppi) [reply README.md:65]: Oh thanks, ill fix it later 👍

**Pull request merged**